repo/packages/C/cassandra/15/package.json

```diff
@@ -5,6 +5,7 @@
   "minDcosReleaseVersion": "1.7",
   "name": "cassandra",
   "packagingVersion": "3.0",
+  "preInstallNotes": "DC/OS Cassandra Service default configuration requires 3 nodes each with 0.3 CPU shares, 1184MB of memory and 272MB of disk.",
```

Comment:
I can change this, but I am not the expert in verifying the accuracy of this statement. I will leave it you / marathon team for the accuracy.

(Pushing an update with the suggested change).